ledgerjs-fio/src/types/public.ts

```diff
@@ -0,0 +1,176 @@
+/**
+ * Type for 64-bit integers.
+ *
+ * We accept either
+ * - `Number` (if it is less than Number.MAX_SAFE_INTEGER)
+ * - `String` (representing 64-bit number)
+ * - `BigInt` (if platform supports it natively)
+ * @category Basic types
+ */
+export type bigint_like = number | bigint | string
+
+// Blockchain-defined constants
+
+/**
+ * Hardened derivation
+ *
+ * @example
+ * ```
+ * const accountId = 0 + HARDENED
+ * ```
+ *
+ * @see [[BIP32Path]]
+ */
+export const HARDENED = 0x80000000
+
+// Our types
+
+/**
+ * Device app flags
+ * @category Basic types
+ * @see [[Version]]
+ */
+export type Flags = {
+    isDebug: boolean
+}
+
+export type DeviceCompatibility = {
```

Comment:
Missing jsdoc annotation:
```suggestion
/**
 * Describes compatibility of device with current SDK
 * @category Basic types
 * @see [[Fio.getVersion]]
 */
export type DeviceCompatibility = {
```